cli.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+print("****************")
```

Comment:
Is it necessary to have these prints before the import section ?

Reply:
Yes, because I wanted to know whether the CLI file is actually running before any error inside it happens.